src/MeshKernelNET/Api/IMeshKernelApi.cs

```diff
@@ -1040,6 +1040,25 @@ public interface IMeshKernelApi : IDisposable
         /// <returns>Error code</returns>
         int Mesh2dInsertEdge(int meshKernelId, int startVertexIndex, int endVertexIndex, ref int edgeIndex);
 
+        /// @brief Insert a new mesh2d edge from 2 coordinates
+        /// <param name="meshKernelId"> The id of the mesh state

```

Comment:
Add an explanation that if the coordinate does not (exactly? within some radius?) match a vertex, then a new one is created.